assets/js/script.js

```diff
@@ -197,4 +200,29 @@ window.addEventListener('load', () => {
       });
     });
   });
+
+  const errorMessage = (message) => {
+    document.getElementById('error').innerHTML = `<p style="padding:10px;">${message}</p>`;
+  };
+  // contact form validation
+  const email = document.getElementById('email');
+  const name = document.getElementById('name');
+  const text = document.getElementById('text');
+  document.querySelector('#submit').addEventListener('click', () => {
+    // if the email field is valid, we let the form submit
+    if (email.validity.typeMismatch) {
+      errorMessage('Invalid Email: Enter a valid email address');
+    } else if (email.validity.patternMismatch) {
+      errorMessage('Please ensure email address is in lowercase');
```

Comment:
- [ OPTIONAL] don't you think it will be better for the error message to appear in Red color. This will enable the user to easily see this as a warning. Nice work.